### PR TITLE
Modal loading component

### PR DESCRIPTION
**Pull request by qmya**
## Goal

Insert purpose of pull request

## Todos:
- [ ] Outstanding todo
- [x] Completed todo
 
## Implementation Decisions

### CODE REVIEW

File: src/components/ModalComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```
Comment (kkyeboah) [inline src/components/ModalComponent.tsx]: Could we build this modal using material UI's modal as a base?

File: src/components/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
```
Comment (kkyeboah) [inline src/components/Button.tsx]: This is an unrelated component which has already been merged.

**Review by kkyeboah — CHANGES_REQUESTED** [review]
Hi @qmya, thanks for getting a PR filed for this issue. 
However, there are a few issues, this PR appears to have unrelated work for a button component. A button component already exists in this repository and can simply be used if that was the intention. Also, there is no example usage in `App.tsx` to demonstrate this modal.

Could you remove the unrelated component and also use `App.tsx` to show how this modal will work? Also, if the intention was to use a button to trigger the modal, a simple link that triggers the modal component on click would be fine here.

File: src/components/ModalComponent.tsx (hunk 1 above)
Comment (qmya) [reply src/components/ModalComponent.tsx]: Thanks, @kkyeboah 🙂  I used material-UI modal this time.

File: src/components/Button.tsx (hunk 2 above)
Comment (qmya) [reply src/components/Button.tsx]: Also deleted the extra button file that I created for the "Parse button" and reused the "ButtonComponent" that is already present in the repo for the "Parse Button".

File: src/components/Modal/Modal.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  onClose: () => void;
+}
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    paper: {
+      width: 350,
+      height:70,
+      boxShadow: theme.shadows[5],
+      padding: theme.spacing(2, 4, 3),
```
Comment (kkyeboah) [inline src/components/Modal/Modal.tsx]: Could we make this properly centered by making the bottom padding the same as the top padding? 
```suggestion
      padding: theme.spacing(2, 4),
```

**Review by kkyeboah — CHANGES_REQUESTED** [review]
Thanks for working on this issue. This looks good, just a single nitpick change request. See comment.

When you make the change, go ahead and also remove your additions to `App.tsx` to make this PR ready to merge.

**Review by kkyeboah — APPROVED** [review]
